script.js

```diff
@@ -1,4 +1,218 @@
+    winningCountComp += 1;
+    result = `computer chose ${randomItem}  💎 <p>and you chose ${input}. YOU win! 🏆. so far, ${userName} won ${winningCountPlayer} number of times.`;
+
+  }
+  else if ((randomItem == 'stone') && (input == 'paper')) {
+    winningCountPlayer += 1;
+    result = `computer chose ${randomItem} 💎 <p>and you chose ${input}. </p>COMPUTER wins! 💻. so far won ${winningCountComp} number of times.</p>`;
+
+  }
+  else if (((randomItem == 'scissors') && (input == 'scissors')) || ((randomItem == 'paper') && (input == 'paper')) || ((randomItem == 'stone') && (input == 'stone'))) {
+    drawCount += 1;
+    result = `computer chose ${randomItem} and you chose ${input}. <p>It's a DRAW!.</p>so far ${drawCount} number of draws.`;
+  }
+  console.log(recentWinner);
+  return result;
+}
+
+// korean version of the game
+function koreanVersion(randomItem, input) {
+  let result;
+  if ((randomItem == 'scissors') && (input == 'paper')) {
+    winningCountComp += 1;
+    result = `computer chose ${randomItem} ✄ <p>and you chose ${input} 📃.</p> COMPUTER wins! 💻. `;
+    recentWinner = 'Computer';
+  }
+  else if ((randomItem == 'scissors') && (input == 'stone')) {
+    winningCountPlayer += 1;
+    result = `computer chose ${randomItem} ✄ <p>and you chose ${input} 💎. YOU win!. 🏆</p> `;
+    recentWinner = 'Player';
+  }
+  else if ((randomItem == 'paper') && (input == 'stone')) {
+    winningCountComp += 1;
+    result = `computer chose ${randomItem} 📃 <p> and you chose ${input} 💎. COMPUTER wins! 💻.`;
+    recentWinner = 'Computer';
+  }
+  else if ((randomItem == 'paper') && (input == 'scissors')) {
+    winningCountPlayer += 1;
+    result = `computer chose ${randomItem} 📃 <p>and you chose ${input} ✄. YOU win! 🏆. </p>`;
+    recentWinner = 'Player';
+  }
+  else if ((randomItem == 'stone') && (input == 'scissors')) {
+    winningCountComp += 1;
+    result = `computer chose ${randomItem}  💎 <p>and you chose ${input}. </p>COMPUTER wins! 💻. `;
+    recentWinner = 'Computer';
+  }
+  else if ((randomItem == 'stone') && (input == 'paper')) {
+    winningCountPlayer += 1;
+    result = `computer chose ${randomItem} 💎 <p>and you chose ${input}. </p>YOU win! 🏆. `;
+    recentWinner = 'Player';
+  }
+  else if (((randomItem == 'scissors') && (input == 'scissors')) || ((randomItem == 'paper') && (input == 'paper')) || ((randomItem == 'stone') && (input == 'stone'))) {
+    drawCount += 1;
+    result = `computer chose ${randomItem} and you chose ${input}. <p>It's a DRAW!.</p>so far ${drawCount} number of draws. The ULTIMATE WINNER is ${recentWinner}.`;
+  }
+  console.log(recentWinner);
+  return result;
+}
+
+// tracking state of the game
+let mode = 'normal';
+
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  let gameResult;
+  if (input == 'reverse') {
+    mode = 'reverse';
+    return gameResult = 'reverse mode set. play the game!'
+  }
+  else if (input == 'normal') {
```

Comment:
No need `else if` if returning from previous `if` statement. ESLint should also tell us this.